apps/src/gamelab/GameLabSprite.js

```diff
@@ -105,6 +108,58 @@ module.exports.createSprite = function (x, y, width, height) {
     }
   });
 
+  Object.defineProperty(s, 'width', {
+    enumerable: true,
+    get: function () {
+      if (width === undefined) {
+        return 100;
+      } else if (!s.animation) {
+        return unscaledWidth;
+      } else {
+        console.log("old width: " + width);
+        return unscaledWidth * this.scale;
+      }
+    },
+    set: function (value) {
+      unscaledWidth = value / this.scale;
```

Comment:
I suspect we ought to have the same dependency on `animation` being set in our setter; if p5play's assumption is that width/height are unscaled when there's no animation, then I suspect setting them with no animation should not go through the inverse scale transform.